### PR TITLE
feat: implement TTS Streaming

### PR DESCRIPTION
**Pull request by wiktor-mazur**
## Describe your changes

Implements TTS Streaming feature.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project and meets the industry quality standards
- [x] I have made corresponding changes to the documentation, `CHANGELOG.md` and `README.md`
- [x] I have updated the package version according to [semantic versioning](https://semver.org/)
- [x] I have provided meaningful commit messages

### CODE REVIEW

File: charactr_api/sdk/tts_stream_simplex.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from typing import Callable
+
+
+class TTSStreamSimplex:
+    def __init__(
+        self,
+        credentials: Credentials,
+        voice_id: int,
+        text: str,
+        blocking: bool = True,
```
Comment (piotrpcharactr) [inline charactr_api/sdk/tts_stream_simplex.py:17]: is there a case, where we would want this to be false? Not sure we need this parametrized.

**Review by piotrpcharactr — APPROVED** [review]

File: charactr_api/sdk/tts_stream_simplex.py (hunk 1 above)
Comment (wiktor-mazur) [reply charactr_api/sdk/tts_stream_simplex.py:17]: I guess someone might not want this function to block execution and handle it themselves by waiting on the `on_closed` callback.

File: charactr_api/sdk/tts.py [hunk 2]
```diff
@@ -5,6 +5,9 @@
 from .data_objects import Audio, Credentials
 from .conversion_module import ConversionModule
 from .errors import get_api_error
+from typing import Callable
```
Comment (mariarozanska) [inline charactr_api/sdk/tts.py:8]: Just a style comment - according to PEP8:

> Imports should be grouped in the following order:
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

File: examples/tts_stream_simplex.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            out_f.write(data)
+
+        def on_close(code: int, message: str) -> None:
+            if code != 1000:
+                raise Exception("Error [{0}]: {1}".format(code, message))
+            else:
+                print("result_tts_stream_simplex.wav has been saved.")
+
+        tts_voices = sdk.tts.get_voices()
+        text = "Hello world from the charactr TTS Duplex Streaming."
```
Comment (mariarozanska) [inline examples/tts_stream_simplex.py:20]: `Duplex` -> `Simplex`

**Review by mariarozanska — APPROVED** [review]

File: charactr_api/sdk/tts.py (hunk 2 above)
Comment (wiktor-mazur) [reply charactr_api/sdk/tts.py:8]: Weird, I thought `black` reorganizes them automatically according to PEP8.

**Pull request merged**